components/script/dom/oscillatornode.rs

```diff
@@ -113,6 +114,24 @@ impl OscillatorNodeMethods for OscillatorNode {
     fn Detune(&self) -> DomRoot<AudioParam> {
         DomRoot::from_ref(&self.detune)
     }
+
+    // https://webaudio.github.io/web-audio-api/#dom-oscillatornode-type
+    fn Type(&self) -> OscillatorType {
+        self.oscillator_type.get()
+    }
+
+    // https://webaudio.github.io/web-audio-api/#dom-oscillatornode-type
+    fn SetType(&self, type_: OscillatorType) -> ErrorResult {
+        self.oscillator_type.set(type_);
+        if type_ == "custom" {
```

Comment:
```
error[E0308]: mismatched types
   --> components/script/dom/oscillatornode.rs:126:21
    |
126 |         if type_ == "custom" {
    |                     ^^^^^^^^ expected enum `dom::bindings::codegen::Bindings::OscillatorNodeBinding::OscillatorType`, found reference
    |
    = note: expected type `dom::bindings::codegen::Bindings::OscillatorNodeBinding::OscillatorType`
               found type `&'static str`
```

WebIDL defined enums generate Rust enums with each entry's first letter capitalized. We need to compare with `OscillatorType::Custom` in this case.